src/ttauri/text/unicode_text_segmentation_tests.cpp

```diff
@@ -15,7 +19,7 @@ using namespace tt;
 
 struct graphemeBreakTest {
     std::u32string codePoints;
-    std::vector<bool> breakOpertunities;
+    std::vector<bool> breakOpportunities;
```

Comment:
Not a blocker.
But if you want to change the spelling, I will also accept it if you change these into snake_case, even if it is just a single variable among many.